### ISSUE TITLE
Error after upgrade to gulp-mocha-phantomjs version 0.10.0 on the windows platform

### ISSUE THREAD

**Issue by doberkofler** [mrhooray/gulp-mocha-phantomjs#46]
The exact same unit tests failed to work after (exclusively) upgrading from version 0.9.0 to 0.10.0.

The error messages looks like this:
```
Error loading resource file://d/MyDev/ljs_app/trunk/periscope/build/test/frontend/lib/runner.html (203).
Details: Error opening //d/MyDev/ljs_app/trunk/periscope/build/test/frontend/lib/runner.html: The network path was not found.
Failed to load the page. Check the url: file://d/MyDev/ljs_app/trunk/periscope/build/test/frontend/lib/runner.html
```

It seems to me, as if the URL would have an invalid format:
The URL `file://d/MyDev/ljs_app/trunk/periscope/build/test/frontend/lib/runner.html` is not a valid URL  on Windows but it should rather (not the `d:` instead of `d`) be `file://d:/MyDev/ljs_app/trunk/periscope/build/test/frontend/lib/runner.html`.

**Comment by mrhooray** [comment]
could you share where does `d/` come from? shouldn't gulp.src return paths with `d:/`?
[this](https://github.com/mrhooray/gulp-mocha-phantomjs/commit/2d280efc6eb3e7458897b9c819be217e118fa289) is the change to 0.10.0

**Comment by doberkofler** [comment]
Yes, the above changes seem to cause this problem. I've added some debug output:
```javascript
function mochaPhantomJS(options) {
    options = options || {};

    var scriptPath = lookup('mocha-phantomjs-core/mocha-phantomjs-core.js');

    if (!scriptPath) {
        throw new gutil.PluginError(pluginName, 'mocha-phantomjs-core.js not found');
    }

    return through.obj(function (file, enc, cb) {
        var args = [
            scriptPath,
            //mergeQuery(toURI(file.path), options.mocha),
            mergeQuery(file.path, options.mocha),
            options.reporter || 'spec',
            JSON.stringify(options.phantomjs || {})
        ];

        console.log('file.path: "' + file.path + '"');
        console.log('toURI(file.path): "' + toURI(file.path) + '"');
        console.log('mergeQuery(file.path, options.mocha): "' + mergeQuery(file.path, options.mocha) + '"');
        console.log('mergeQuery(toURI(file.path), options.mocha): "' + mergeQuery(toURI(file.path), options.mocha) + '"');

        spawnPhantomJS(args, options, this, function (err) {
            if (err) {
                return cb(err);
            }

            this.push(file);

            cb();
        }.bind(this));
    });
}
```
to show what happens:
```
file.path: "D:\MyDev\ljs_app\trunk\periscope\build\test\frontend\onerror\runner.html"
toURI(file.path): "file://D:\MyDev\ljs_app\trunk\periscope\build\test\frontend\onerror\runner.html"
mergeQuery(file.path, options.mocha): "d:/MyDev/ljs_app/trunk/periscope/build/test/frontend/onerror/runner.html"
mergeQuery(toURI(file.path), options.mocha): "file://d/MyDev/ljs_app/trunk/periscope/build/test/frontend/onerror/runner.html"
```

**Comment by mrhooray** [comment]
@doberkofler could you test with latest master?

**Comment by doberkofler** [comment]
@mrhooray Latest master does work again!

**Comment by mrhooray** [comment]
published v0.10.1
